packages/react-devtools-shared/src/devtools/views/Components/InspectedElementErrorsAndWarningsTree.css

```diff
@@ -1,51 +1,36 @@
+.ErrorTree {
+  background-color: var(--color-error-background-message);
+  color: var(--color-error-text);
+}
+
+.WarningTree {
+  background-color: var(--color-warning-background);
+  color: var(--color-warning-text);
+}
+
 .ErrorOrWarning {
   align-items: center;
   border-bottom: 1px solid transparent;
   display: flex;
   padding-left: 1rem;
 }
 
-.ErrorOrWarningMessage {
-  overflow-wrap: break-word;
-  white-space: pre-wrap;
+.Error, .Warning {
+  padding: 0 0.25rem;
+  display: flex;
+  align-items: center;
 }
 
 .Error {
-  background-color: var(--color-error-background-message);
```

Comment:
The idea behind the background color was to ease vertical scanning for errors/warnings. Right now you have to scan vertically and look at the end of each row. It's barely visible in light mode but in dark mode it really helped scanning. Though it might not be necessary with the "scroll to next error" UX. Will keep an eye on it when using it locally.

Reply:
Yeah. I was concerned about too many background colors getting confusing. I also didn't think it was totally necessary with the prev/next step controls.